homeassistant/components/hive/translations/en.json

```diff
@@ -34,7 +41,7 @@
                     "scan_interval": "Scan Interval (seconds)",
                     "username": "Username"
                 },
-                "description": "Enter your Hive login information and configuration.",
```

Comment:
Does this change also need to be made in strings.json?

Reply:
Done